op-node/flags/flags.go

```diff
@@ -57,6 +69,30 @@ var (
 		EnvVars: prefixEnvVars("RPC_PORT"),
 		Value:   9545, // Note: op-service/rpc/cli.go uses 8545 as the default.
 	}
+	RPCListenReadTimeout = &cli.DurationFlag{
```

Comment:
What's the benefit to having so many configuration options here? Are these set by default

Reply:
Yes, these values are set to defaults. However, the default timeout may be insufficient for RPC requests that take a long time. For instance, if the `eth_getProof` request exceeds 30 seconds, the `op-node`'s response to `optimism_outputAtBlock` will result in an error.